.gitignore

```diff
@@ -14,3 +14,10 @@
 /denoDist
 /npm
 /deno
+/build
+.docusaurus
+.cache-loader
+.DS_Store
+npm-debug.log*
+yarn-debug.log*
```

Comment:
can you please leave only the stuff that docusaurus produces?

Reply:
`.DS_Store` was getting added locally so I added these gitignores which imo should exist see: https://github.com/github/gitignore/blob/master/Node.gitignore